modules/sdpolicy-platform.adoc

```diff
@@ -30,15 +29,15 @@ The following table shows the frequency of backups:
 |Retention
 |Notes
 
-|Full object store backup, all cluster PVs
+|Full object store backup
 |Daily at 0100 UTC
 |7 days
-|This is a full backup of all Kubernetes objects, as well as all mounted PVs in the cluster.
+|This is a full backup of all Kubernetes objects. No PVs are backed up in this backup schedule.
```

Comment:
You haven't spelled out PV yet, so here I would write, "No persistent volumes (PVs) are backed up in this backup schedule."

Reply:
Resolved